types/src/system/auction/bid.rs

```diff
@@ -129,7 +143,7 @@ impl Bid {
         &self.staked_amount
     }
 
-    /// Gets the staked amount of the provided bid
+    /// Gets the staked amount of the provided bidshould_release_vfta_holder_stake
```

Comment:
```suggestion
    /// Gets the staked amount of the provided bid should_release_vfta_holder_stake
```

Reply:
/// Gets the staked amount of the provided bid